### PR TITLE
Update Java 21 test to use GA version

### PR DESCRIPTION
**Pull request by gjwatts**
Switch building the Java 21 WAR file from Java 21 EA to GA

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ name: Java CI with Gradle
 on:
   push:
     branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```
Comment (KyleAure) [inline .github/workflows/CI.yml:7]: Perhaps if we are going to run this build on PRs then we should add the following to the `Upload Applciation` step: 
```yml
if: github.event_name == 'push' && github.ref == 'refs/heads/main'
```

That way we only upload WAR files after something has been merged to main, and we can save space.

File: .gitignore [hunk 2]
```diff
@@ -41,3 +41,7 @@ hs_err_pid*
 # MacOS Garbage
 .DS_Store
 /*/.DS_Store
+
+.classpath
+.project
+*/.settings
```
Comment (KyleAure) [inline .gitignore:47]: If you are going to ignore these files, then you should likely delete the existing files from github.

**Review by KyleAure — APPROVED** [review]
Looks good just a couple comments.

File: .gitignore (hunk 2 above)
Comment (gjwatts) [reply .gitignore:47]: Good idea -> https://github.com/OpenLiberty/open-liberty-misc/pull/13

File: .github/workflows/CI.yml (hunk 1 above)
Comment (gjwatts) [reply .github/workflows/CI.yml:7]: Done

**Pull request merged**